cocotb/handle.py

```diff
@@ -438,10 +450,19 @@ def __hash__(self):
         return SimHandleBase.__hash__(self)
 
 class ConstantObject(NonHierarchyObject):
-    """Constant objects have a value that can be read, but not set.
+    """A constant object.
+
+    Constant objects have a value that can be read, but not set.
+
+    We can also cache the value since it is fixed at elaboration time and
+    won't change within a simulation.
 
-    We can also cache the value since it is elaboration time fixed and won't
-    change within a simulation.
+    Args:
+        handle (int): The GPI handle to the simulator object.
+        path (str): Path to this handle, ``None`` if root.
+        handle_type: The type of the handle 
+            (``simulator.INTEGER``, ``simulator.ENUM``, 
+            ``simulator.REAL``, ``simulator.STRING``).
```

Comment:
Aside: It's really tempting to take a dependency on `enum34` for python2, so we can clean up a bunch of these into real enum types.

Reply:
In a new issue, yes :)